ribbon/ribbon/swap.py

```diff
@@ -21,6 +21,21 @@
 from ribbon.encode import ADDRESS_ZERO
 from shutil import ExecError
 
+# ---------------------------------------------------------------------------
+# Constants
+# ---------------------------------------------------------------------------
+DETAILED_ERROR_MESSAGES = {
+    "SIGNATURE_INVALID": "Signature invalid.",
+    "SIGNATURE_MISMATCHED": "Signature's origin does not match signer's address. Ensure you are using the correct wallet.",
+    "NONCE_ALREADY_USED": "Nonce has been used.",
+    "BID_TOO_SMALL": "Bid size has to be larger than minimum bid.",
+    "BID_EXCEED_AVAILABLE_SIZE": "Bid size has to be smaller than available size.",
+    "PRICE_TOO_LOW": "Price per oToken has to be larger than minimum price.",
+    "SIGNER_ALLOWANCE_LOW": "Insufficient bidding token allowance. Ensure you have approved sufficient amount of bidding token.",
+    "SIGNER_BALANCE_LOW": "Insufficient bidding token balance. Ensure you have sufficient balance of bidding token in your wallet.",
+    "SELLER_ALLOWANCE_LOW": "Seller has insufficient oToken allowance.",
```

Comment:
About these last two, which are related to the seller, so Ribbon, should we really show this error to users?
Probably only if the maker is belonging to Ribbon? 
Open to more thoughts.

Reply:
Sure, we can just not show the error for the last 2. They pretty much won't happen to our vaults because the approval and balance calculations are automated on the vaults.

Reply:
Ok, good to know. I'll keep a note for the team 🙏🏽